plugins/jobs/server/models/job.py

```diff
@@ -28,19 +28,23 @@
 
 
 class Job(AccessControlledModel):
+
     def initialize(self):
         self.name = 'job'
         compoundSearchIndex = (
             ('userId', SortDir.ASCENDING),
-            ('created', SortDir.DESCENDING)
+            ('created', SortDir.DESCENDING),
+            ('type', SortDir.ASCENDING),
+            ('status', SortDir.ASCENDING)
         )
         self.ensureIndices([(compoundSearchIndex, {}), 'created'])
 
         self.exposeFields(level=AccessType.READ, fields={
             'title', 'type', 'created', 'interval', 'when', 'status',
             'progress', 'log', 'meta', '_id', 'public', 'async', 'updated', 'timestamps'})
 
-        self.exposeFields(level=AccessType.SITE_ADMIN, fields={'args', 'kwargs'})
+        self.exposeFields(level=AccessType.SITE_ADMIN,
```

Comment:
@danlamanna Thanks for the tip 👍, but seems the formatter I use doesn't support flake8 setting, but I'll keep the flake8.cfg in mind in future.